src/backend/tests/risks.test.ts

```diff
@@ -144,5 +144,33 @@ describe('Risks', () => {
       expect(prisma.risk.findUnique).toHaveBeenCalledTimes(1);
       expect(prisma.risk.update).toHaveBeenCalledTimes(0);
     });
+
+    test('the user does not have permissions to create the risk', async () => {
```

Comment:
right now these are inside the `editRisk` `describe` block. can you make a new describe called `createRisk` and put these there

Reply:
Fixed this!